compute_endpoint/globus_compute_endpoint/endpoint/config/config.py

```diff
@@ -1,221 +1,220 @@
 
-    force_mu_allow_same_user : bool
-        If set, override the heuristic that determines whether the uid running the
-        multi-user endpoint may also run single-user endpoints.
+    :param idle_heartbeats_hard: Number of heartbeats after no task or result has moved
+        before the endpoint shuts down.  Unlike `idle_heartbeats_soft`, this idle timer
+        does not require that there are no outstanding tasks or results.  If no task or
+        result has moved in this many heartbeats, then the endpoint will shut down.  In
+        particular, this is intended to catch the error condition that a worker has
+        gone missing, and will thus _never_ return the task it was sent.  Note that
+        this setting is only enabled if the `idle_heartbeats_soft` is a value greater
+        than 0.  Suggested value: a multiplier of heartbeat_period equivalent to two
+        days.  For example, if `heartbeat_period` is 30s, then suggest 5760.
 
-        Normally, the multi-user endpoint disallows starting single-user endpoints with
-        the same UID as the parent process unless the UID has no privileges.  That
-        means that the UID is not 0 (root), or that the UID does *not* have (among
-        many others) the capability to change the user (otherwise known as "drop
-        privileges").
+    :param detach_endpoint: Whether the endpoint daemon be run as a detached process.
+        This is good for a real edge node, but an anti-pattern for kubernetes pods
 
-        Default: False
+    :param endpoint_setup: Command(s) to be run during the endpoint initialization
+        process
 
-    display_name : str | None
-        The display name for the endpoint.  If None, defaults to name
-        Default: None
+    :param endpoint_teardown: Command(s) to be run during the endpoint
+        shutdown process
 
-    endpoint_setup : str | None
-        Command or commands to be run during the endpoint initialization process.
+    :param log_dir: (Optional) path to the top-level directory where logs should be
+        written
 
-    endpoint_teardown : str | None
-        Command or commands to be run during the endpoint shutdown process.
+    :param stdout: Path where the endpoint's stdout should be written
 
-    mu_child_ep_grace_period_s : float
-        If a single-user endpoint dies, and then the multi-user endpoint receives a
-        start command for that single-user endpoint within this timeframe, the
-        single-user endpoint is started back up again.
-        Default: 30 seconds
+    :param stderr: Path where the endpoint's stderr should be written
 
-    amqp_port : int | None
-        Port to use for AMQP connections. Note that only 5671, 5672, and 443 are
-        supported by the Compute web services. If None, the port is assigned by the
-        services (which default to 443).
-        Default: None
     """
 
     def __init__(
         self,
+        *,
         # Execution backed
```

Comment:
Damnit; I had corrected that in one iteration (because it exists in release code ... that no user sees!), but that got lost somewhere.  Thanks for pointing it out.